scripts/install.sh

```
@@ -112,33 +112,51 @@ install_dependencies() {
+  RUST_MD5_URL="{{.RustDevMd5URL}}"
+{{else}}
+  OS="linux"
+  # Matches the rust release asset: the large-disk variant tarball holds a
+  # binary named weed-volume-large-disk. OSS publishes weed-volume_... from
+  # seaweedfs/seaweedfs; enterprise publishes weed-volume-enterprise_... from
+  # seaweedfs/artifactory (ReleaseOwner/ReleaseRepo already select the repo).
+  RUST_VERSION_ID="${SEAWEED_VERSION}"
+  RUST_ASSET="weed-volume{{if .Enterprise}}-enterprise{{end}}_large_disk_${OS}_${SUFFIX}.tar.gz"
+  RUST_URL="https://github.com/{{.ReleaseOwner}}/{{.ReleaseRepo}}/releases/download/${SEAWEED_VERSION}/${RUST_ASSET}"
+  RUST_MD5_URL="${RUST_URL}.md5"
+{{end}}
```

Shellcheck (0.11.0)

[error] You need a space after the '{'.

(SC1054)